src/views/ApartmentApp/components/StudentApplication/components/OffCampusSection/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import { Grid, Card, CardHeader, CardContent, List, Button } from '@material-ui/core';
+import AddIcon from '@material-ui/icons/Add';
+import ProgramListItem from './components/ProgramListItem';
+import goStalk from '../../../../../../services/goStalk';
+
+// Create a list of selection boxes to choosing which applicants are doing off campus programs.
+const OffCampusSection = ({
+  offCampusApplicantList,
+  availableApplicants,
+  onOffCampusChanged,
+  onOffCampusApplicantAdd,
+  onOffCampusApplicantRemove
+}) => {
+  const [availableMajors, setAvailableMajors] = useState([]);
+
+  useEffect(() => {
+    const loadDepartments = async () => {
+      let unfilteredDepartments;
+      try {
+        // Get the majors that the applicants may be completing off campus programs for
+        unfilteredDepartments = await goStalk.getMajors();
+      }
+      catch {
+        // Get the majors that the applicants may be completing off campus programs for
+        unfilteredDepartments = await goStalk.getMajors();
+      }
+      //Remove spaces from strings
+      setAvailableMajors(unfilteredDepartments.map((major) => major.trim()));
+    };
+
+    loadDepartments();
+  });
+
+  const handleInputChange = (ApplicantNameValue, ApplicantMajorValue, index) => {
+    onOffCampusChanged(ApplicantNameValue, ApplicantMajorValue, index);
+  };
+
+  const handleRemove = (index) => {
+    // Make sure the chosen index was not null
+    if (index !== null) {
+      // Send the selected index to the parent component
+      onOffCampusApplicantRemove(index);
+    }
+  };
+
+  const handleAddDropdown = () => {
+    onOffCampusApplicantAdd();
+  };
+
+  return (
+    <Card>
+      <CardHeader title="Off Campus Program Applicants" className="card-header" />
+      <CardContent>
+        <Grid container justify="space-between" spacing={2}>
+          <Grid item xs={12}>
+            <List className="off-Campus-list" aria-label="apartment applicants off campus programs" disablePadding>
+              {offCampusApplicantList ? (
+                offCampusApplicantList.map((memberInfo, index) => (
```

Comment:
You could easily use an array map on the original applicants array rather than creating this additional array with duplicate information. Either way, you have to make sure that the state variable `applicants` in `StudentApplication/index.js` is being updated to include the off campus selection, which does not appear to be the case